config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -316,7 +316,7 @@ presubmits:
         - name: RUNTIME_CONFIG
           value: '{"api/alpha":"true", "api/ga":"true"}'
         - name: LABEL_FILTER
-          value: "!Slow && !Disruptive && !Flaky && Feature: isSubsetOf Alpha"
+          value: "Alpha && !BetaOffByDefault && Feature: isSubsetOf OffByDefault && !Slow && !Disruptive && !Flaky"
```

Comment:
NOTE: this is more verbose now, but the previous filter did not permit "alpha feature that depends on an on-by-default beta feature" despite the cluster being capable.

Reply:
Do we want to rename the jobs now? We were considering calling this "alpha-tests".

As it stands now, the description no longer matches and the job no longer does what it was originally meant to do (prevent regressions in GA features caused by enabled alphas). We need to add "alpha-cluster" or "alpha-enabled".

Reply:
I think we should do that later so we can bikeshed after code freeze, this is pretty close to what the job was doing before this PR?

Renaming jobs is a bit messy and I want to minimize surprises during this window (e.g. if we rename the presubmit then people who are using it to test in-flight PRs during this high volume may be confused, we can do more communication when people are not busy with code freeze or kubecon)

Reply:
> this is pretty close to what the job was doing before this PR

No, it does the opposite: previously it checked for regressions of GA features, now it runs alpha tests.

> Renaming jobs is a bit messy and I want to minimize surprises during this window

Then let's keep the "-features" name and previous behavior (aka "-cluster" or "-enabled") and add new "-test" jobs.

Reply:
> No, it does the opposite: previously it checked for regressions of GA features, now it runs alpha tests.

Er, previously it ran `Feature: isSubsetOf Alpha` ...? (ignoring flaky/slow/disruptive)

Reply:
It also ran tests for GA features - that was and is its primary purpose.

Reply:
/hold
this needs fixing, and the other kind jobs need reviewing. I'll work on that shortly